modules/notify/lib/notify/service.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'notifications/client'
+require 'common/client/base'
+require_relative 'configuration'
+
+module Notify
+  class Service < Common::Client::Base
+    include Common::Client::Concerns::Monitoring
+
+    STATSD_KEY_PREFIX = 'api.vanotify'
+
+    configuration VaNotify::Configuration
+
+    attr_reader :notify_client
+
+    # rubocop:disable Lint/MissingSuper
```

Comment:
Is there a way around this without disabling inline? For example, I see that [this service class ](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/meb_api/lib/dgi/service.rb)in a module didn't have the same call to super linting issue

Reply:
Nvm ignore me! It looks like we're adding the cop disabling to the .rubocop_todo.yml

Reply:
@nathanbwright You'll just have to update the rubocop_todo! Thanks!